ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -5485,112 +5593,6 @@ protected override void BuildModel(ModelBuilder modelBuilder)
-                                .HasColumnType("int");
-
-                            b1.Property<string>("Address")
-                                .HasMaxLength(150)
-                                .HasColumnType("nvarchar(150)");
-
-                            b1.Property<int?>("CountryId")
-                                .HasColumnType("int");
-
-                            b1.Property<DateTime?>("Dob")
-                                .HasColumnType("datetime2");
-
-                            b1.Property<int?>("EthnicityId")
-                                .HasColumnType("int");
-
-                            b1.Property<string>("FamilyName")
-                                .HasMaxLength(35)
-                                .HasColumnType("nvarchar(35)");
-
-                            b1.Property<string>("GivenName")
-                                .HasMaxLength(35)
-                                .HasColumnType("nvarchar(35)");
-
-                            b1.Property<string>("LocalPatientId")
-                                .HasMaxLength(50)
-                                .HasColumnType("nvarchar(50)");
-
-                            b1.Property<string>("NhsNumber")
-                                .HasColumnType("nvarchar(max)");
-
-                            b1.Property<bool>("NhsNumberNotKnown")
-                                .HasColumnType("bit");
-
-                            b1.Property<bool>("NoFixedAbode")
-                                .HasColumnType("bit");
-
-                            b1.Property<int?>("OccupationId")
-                                .HasColumnType("int");
-
-                            b1.Property<string>("OccupationOther")
-                                .HasMaxLength(150)
-                                .HasColumnType("nvarchar(150)");
-
-                            b1.Property<string>("Postcode")
-                                .HasColumnType("nvarchar(max)");
-
-                            b1.Property<string>("PostcodeToLookup")
-                                .HasColumnType("nvarchar(10)");
-
-                            b1.Property<int?>("SexId")
-                                .HasColumnType("int");
-
-                            b1.Property<int?>("YearOfUkEntry")
-                                .HasColumnType("int");
-
-                            b1.HasKey("NotificationId");
-
-                            b1.HasIndex("CountryId");
-
-                            b1.HasIndex("EthnicityId");
-
-                            b1.HasIndex("OccupationId");
-
-                            b1.HasIndex("PostcodeToLookup");
-
-                            b1.HasIndex("SexId");
-
-                            b1.ToTable("Patients");
-
-                            b1.HasOne("ntbs_service.Models.ReferenceEntities.Country", "Country")
-                                .WithMany()
-                                .HasForeignKey("CountryId");
```

Comment:
The patient details used to "have" some of these properties: country, ethnicity, ect. In the new code that is in this file for patient details that doesn't seem to be the case - any idea why?

Reply:
No, I don't. I agree that it seems strange, but (assuming that there are no problems) I don't think it's worth investigating this further.